src/main/java/com/duckblade/osrs/dpscalc/calc/gearbonus/VampyreGearBonus.java

```diff
@@ -1,33 +1,59 @@
 package com.duckblade.osrs.dpscalc.calc.gearbonus;
 
+import com.duckblade.osrs.dpscalc.calc.WeaponComputable;
 import com.duckblade.osrs.dpscalc.calc.compute.ComputeContext;
 import com.duckblade.osrs.dpscalc.calc.compute.ComputeInputs;
+import com.duckblade.osrs.dpscalc.calc.model.DefenderAttributes;
 import com.duckblade.osrs.dpscalc.calc.model.GearBonuses;
+import com.google.common.collect.ImmutableMap;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.ItemID;
+
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import lombok.RequiredArgsConstructor;
+import java.util.Map;
+
+import static com.duckblade.osrs.dpscalc.calc.compute.ComputeInputs.ATTACK_STYLE;
 
-// TODO
 @Singleton
 @RequiredArgsConstructor(onConstructor = @__(@Inject))
 public class VampyreGearBonus implements GearBonusComputable
 {
 
+	private static final Map<Integer, GearBonuses> weaponToBonus = ImmutableMap.of(
+			ItemID.BLISTERWOOD_FLAIL, GearBonuses.of(1.05, 1.25),
+			ItemID.IVANDIS_FLAIL, GearBonuses.of(1, 1.20)
```

Comment:
The blisterwood sickle also provides `Gearbonuses.of(1.05, 1.15)`

https://oldschool.runescape.wiki/w/Blisterwood_sickle